public/scripts/plugins/pcr/views/pcr_form_view.js

```diff
@@ -45,9 +51,24 @@ export default Backbone.View.extend({
     this.hasRdpPcrSequence = !this.hasRdpOligoSequence;
     var partType = this.model.get('partType');
 
+    var tryShowingModalKey = 'tryShowingModal';
```

Comment:
This line is throwing out an 'invalid token' through eslint for me.

It builds and works, and I can't see anything wrong with the block so I'm fairly sure it's a false positive.